### PR TITLE
#166789945 Setup React Redux

### PR DESCRIPTION
**Pull request by kodek-sleuth**
#### What does this PR do?
Should setup redux 
#### Description of Task to be completed
- Setup redux
#### How should this be manually tested?
```
Run the following commands in your terminal after cloning the project
1. cd ah-92explorers-frontend
2. git checkout ch-setup-redux-166789945
3. `npm i` to install all dependencies 
4. `npm run dev` and navigate to localhost:8080
```
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#166789945](https://www.pivotaltracker.com/story/show/166789945)
#### Checklist

- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I don't have more than 3 major commits on this PR

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
-    <div class="container">
-        <div id="root">
-        </div>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css?family=Roboto|Roboto+Slab&display=swap" rel="stylesheet">
+    <link rel=”icon” href=”public/favicon.ico”>
+    <title>React App</title>
```
Comment (elemanhillary-zz) [inline public/index.html]: title `Authors Haven` not `React App`

File: src/components/Routes.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+          <Fragment>
+            <Route exact path='/' component={Home} />
+          </Fragment>
+        </Switch>
+     </Router>
+    );
+  }
+}
+
+export default Routes;
```
Comment (elemanhillary-zz) [inline src/components/Routes.js]: new line at EOF

**Review by elemanhillary-zz — CHANGES_REQUESTED** [review]
kindly put back the linting scripts in `package.json` and fix all new line at EOF

**Review by IsaiahRn — COMMENTED** [review]
Kindly, apply the changes requested by hillary?

**Review by Hervera — CHANGES_REQUESTED** [review]
Add a new blank line in each file

**Review by niyoceles — APPROVED** [review]

**Review by Hervera — APPROVED** [review]
Ready to be merged

**Review by IsaiahRn — APPROVED** [review]

File: .babelrc [hunk 3]
```diff
@@ -1,3 +1,8 @@
 {
-  "presets": ["@babel/preset-env", "@babel/preset-react"]
+    "presets": ["@babel/preset-env", "@babel/preset-react"],
+    "plugins": [
+        [
+          "@babel/plugin-proposal-class-properties"
```
Comment (kipropbrian) [inline .babelrc]: Fix this, make it not an array. In future we can revert back.

**Review by elemanhillary-zz — CHANGES_REQUESTED** [review]
kindly implement the project structure in your project we had earlier agreed on. Every component under the component folder should have its own folder. Thanks

**Review by niyoceles — APPROVED** [review]

**Review by IsaiahRn — APPROVED** [review]

**Review by Hervera — APPROVED** [review]

**Pull request merged**